ocw-cli/cli_app.py

```diff
@@ -1232,11 +1180,11 @@ def settings_screen(header):
          if option == '4':
               screen.addstr(25, x/2, "Enter Spatial Gridding Option [Model, Observation or User]:")
               new_spatial_grid_option = screen.getstr()
-              if new_spatial_grid_option.lower() == 'model':
-                   spatial_grid_option = 'Model'
-                   spatial_grid_setting = model_lat_res
-                   note = "Spatial gridding option has changed successfully to {0}".format(spatial_grid_option)
-              elif new_spatial_grid_option.lower() == 'observation':
+              #if new_spatial_grid_option.lower() == 'model':
```

Comment:
Well, considering limited functionality of CLI, we would rather remove the option of regridding datasets into model grid points. If we load multiple model datasets, which model grid does a user choose for regridding? User specified grid points can be used if a user still wants to regrid data into a specific model's grid points.